universitat-freiburg-geschichte.csl

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="note" default-locale="de-DE" version="1.0" demote-non-dropping-particle="sort-only">
   <info>
-    <title>Universität Freiburg - Geschichte (German)</title>
+    <title>Albert Ludwigs Universität Freiburg - Geschichte (German)</title>
```

Comment:
But to be nitpicky, I think the name should be "Albert-Ludwigs-Universität Freiburg" (with hyphens).